processAndroid.js

```diff
@@ -0,0 +1,176 @@
+import StyleDictionary from "style-dictionary";
+import fs from "fs";
+import fsExtra from "fs-extra";
+import path from "path";
+import "./lib/android/transform.js";
+import "./lib/android/format.js";
+import {
+  extractTokens,
+  toCamelCase,
+  transformValue,
+  getGeneratedDate,
+} from "./lib/android/helpers.js";
```

Comment:
These helpers don't seem to be used anywhere in this file. Actually `extractTokens` and `transformValue` isn't used anywhere as far as I checked.